src/molecules/formfields/Select/BasicSelectField/index.js

```diff
@@ -0,0 +1,106 @@
+import React, { PropTypes } from 'react';
+import classnames from 'classnames';
+
+import styles from 'molecules/formfields/shared/formfields.module.scss';
+import renderPlaceholder from 'utils/Fields/renderPlaceholder';
+import renderOptGroup from 'utils/Fields/renderOptGroup';
+
+function BasicSelectField ( props ) {
+  const {
+    className,
+    type,
+    placeholder,
+    label,
+    selectOptions,
+    forProp,
+    focused,
+    onChange,
+    value
+  } = props;
+
+  return (
+    <div className={ classnames(className, styles.selectfield, { [styles.focused]: focused } ) }>
```

Comment:
The `className` here needs to be the last param in the `classnames()` param list